rules/network.js

```diff
@@ -75,19 +75,25 @@ function _system_update_ip(name, iface) {
 };
 
 function _current_active_connection() {
-  runShellCommand('ip route get {} | grep -oP \'dev\\s+\\K[^ ]+\''.format(checkAddress), {
+  runShellCommand('ip route get {} 2>/dev/null | grep -oP \'dev\\s+\\K[^ ]+\''.format(checkAddress), {
     captureOutput: true,
     exitCallback: function (exitCode, capturedOutput) {
-      if (exitCode === 0) {
-        dev.network["Default Interface"] = capturedOutput;
-      }
+      dev.network["Default Interface"] = exitCode === 0 ? capturedOutput : "";
```

Comment:
```suggestion
      dev.network["Default Interface"] = exitCode === 0 ? capturedOutput.trim() : "";
```

Без этого Active Connection у меня не заработало, кажется, там `\n` был лишним

Reply:
поправил